docs/public-networks/how-to/pre-merge-history-expiry.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Prune pre-merge history
+sidebar_position: 9
+description: Reduce the size of your database by removing pre-merge blocks from your blockchain
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Node operators using [Snap sync](../concepts/node-sync.md#snap-synchronization) can significantly reduce
+disk usage by removing [pre-merge](https://ethereum.org/en/roadmap/merge/) Proof of Work (PoW)
+block data from the local database.
+
+Besu can prune all pre-merge blocks and associated transaction receipts, leaving only headers and
+the genesis block.
+
+:::warning
+
+Besu does not currently provide a way to manually import pre-merge block data after pruning.
+If you need to restore the full pre-merge history, you can revert to the old sync method and download
+all blocks from peers by setting
+[`--snapsync-synchronizer-pre-checkpoint-headers-only-enabled=false`](../reference/cli/options.md#snapsync-synchronizer-pre-checkpoint-headers-only-enabled).
+
+Support for Era1 sync for archive nodes is currently in progress.
+
+:::
+
+Besu provides multiple options to prune the historical blockchain data:
+- [Offline pruning](#offline-pruning) to prune data on a stopped Besu instance.
+- [Online pruning](#online-pruning) to prune data on a running Besu instance.
+- [Sync Besu](#sync-without-pre-merge-blocks) using Snap sync, which by default prunes the historical
+    blockchain data.
+
+## Offline pruning
+
+The easiest and fastest option for pruning pre-merge blocks is to perform an offline prune.
+
+1. Ensure your Besu instance has stopped, and run the following command:
+
+    ```bash
+    besu --network=mainnet --data-path=/path/to/your/database storage prune-pre-merge-blocks --threads=4 --prune-range-size=12000
+    ```
+    In the command:
+    - [`--threads`](../reference/cli/subcommands.md#prune-pre-merge-blocks) sets the number of processors
+        to use during the pruning process. This CLI option defaults to
+        one less than the number of processors available on your system, so it's safe to omit.
+    - [`--prune-range-size`](../reference/cli/subcommands.md#prune-pre-merge-blocks) specifies the size of
+        block ranges to be pruned. The default is `10000` and can be adjusted if desired, although
+        this may impact pruning performance.
+
+    On completion, you'll receive the `Pruning pre-merge blocks and transaction receipts completed` log message.
+
+1. Restart Besu and include the [`--history-expiry-prune`](../reference/cli/options.md#history-expiry-prune)
+    option to reclaim the space.
```

Comment:
In testing, we saw the space increases by up to 200GB before the space was finally reclaimed. Probably worth a mention.